src/paperqa/docs.py

```diff
@@ -395,7 +395,7 @@ async def aadd(  # noqa: PLR0912
         # loose check to see if document was loaded
         if metadata.parse_type != "image" and (
             not texts
-            or len(texts[0].text) < 10  # noqa: PLR2004
+            or len(texts[0].text.replace("\n", "")) < 20  # noqa: PLR2004
```

Comment:
Can you add a comment describing what motivates 20, and why we care that there are at least that many non-newline characters?